src/main/webapp/EditProfilePage.html

```diff
@@ -2,15 +2,16 @@
 <html>
   <body>
     <h1>Edit Your Profile Page</h1>
+    <!--TODO(oyins): Make sure form submission edits Datastore-->
     <form action="/profile-data" method="POST">
       <label for="name">Name:</label><br />
-      <input type="text" id="name" name="name" /><br />
+      <input type="text" name="name" /><br />
       <label for="email">Email:</label><br />
-      <input type="email" id="email" name="email" /><br />
+      <input type="email" name="email" /><br />
       <label for="username">Username:</label><br />
-      <input type="text" id="username" name="username" /><br />
```

Comment:
You should probably set a max limit of characters for input such as username and bio so that it's not possible to add millions of characters. HTML has a maxlength attribute for input tags(https://www.w3schools.com/tags/att_input_maxlength.asp).